packages/admin-panel/src/editor/actions.js

```diff
@@ -43,6 +43,12 @@ export const openBulkEditModal =
           type: EDITOR_DATA_FETCH_SUCCESS,
           recordData: response.body,
         });
+        dispatch({
```

Comment:
Does this relate to this PR? I can't see how it fits in?

Reply:
Yes, sorry, there was an overhang within the commit. Ignore this.